urlchecker/core/urlproc.py

```diff
@@ -229,6 +230,13 @@ def check_urls(
 
                 # If we try again, pause for retry seconds and update retry seconds
                 if rcount > 0 and do_retry:
+
+                    # If we've failed once, try adding the accept header
+                    headers = {
+                        "User-Agent": get_user_agent(),
+                        "Accept": "application/json",
+                    }
```

Comment:
btw why did you choose to include the Accept argument only on retry?  Is it faster this way ?

Reply:
No my concern is that since we are emulating a browser visit more pages will reject having this header. This is a compromise then try it only given the first approach doesn’t work.

do you know a good source of recent user agents?

Reply:
sounds good to me. 
Unfortunately, I don't know that much about user agents :/ matter of fact I am like Jon Snow when it comes to them ... I know nothing :p So I will need to read more on them before giving a helpful opinion.

Reply:
I found a solution for this - #76 !